### ISSUE TITLE
GFT sensor shows "Geen", day after tomorrow shows GFT

### ISSUE THREAD

**Issue by tomlevels** [xirixiz/Home-Assistant-Sensor-Afvalwijzer#58]
The sensor trash.gft has value "Geen", but the trash_day_after_tomorrow shows "gft"
![image](https://user-images.githubusercontent.com/3691709/80541183-ff5dbb80-89aa-11ea-957a-1b853e4543ab.png)
![image](https://user-images.githubusercontent.com/3691709/80541316-3af88580-89ab-11ea-9035-3e0f560333fb.png)

I see that the friendly name contains a space, maybe that the previous issue with the space is not completely fixed?
friendly_name: 'trash_gft '

**Comment by xirixiz** [comment]
This is something you configure yourself and isn't an issue with the component. Check the names used for your images, and maybe check this as a reference:

https://github.com/xirixiz/my-hass-config/blob/master/packages/trash.yaml

**Comment by tomlevels** [comment]
But as far as I know, I do not create the sensor and the sensor shows "Geen", but it should show a date?

**Comment by xirixiz** [comment]
That's certainly true :). However, Geen is default when no data is retrieved for that sensor. Can you share a postal code and housenumber matching you area? Or maybe you can check yourself:

https://json.mijnafvalwijzer.nl/?method=postcodecheck&postcode=&huisnummer=&toevoeging=&langs=nl

Should match the following....

https://json.mijnafvalwijzer.nl/?method=postcodecheck&postcode=1111AA&huisnummer=11&toevoeging=A&langs=nl

Check what the upcoming GFT pickup says in json.

**Comment by tomlevels** [comment]
An example postal code is 6851GJ number 2. I tried your link but the json contains a huge amount of info

**Comment by xirixiz** [comment]
I think there's still something going wrong in your code:

When I execute the code manually:
```
Afvaldienst('mijnafvalwijzer', '6851GJ', '2', '', 'false')
```

I get the following output:

```
[{'key': 'pmd', 'days_remaining': 0, 'value': '29-04-2020'}, {'key': 'gft', 'days_remaining': 1, 'value': '30-04-2020'}, {'key': 'papier', 'days_remaining': 5, 'value': '04-05-2020'}, {'key': 'restafval', 'days_remaining': 19, 'value': '
18-05-2020'}]
[{'key': 'first_next_in_days', 'value': 1}]
[{'key': 'today', 'value': 'pmd'}]
[{'key': 'tomorrow', 'value': 'gft'}]
[{'key': 'day_after_tomorrow', 'value': 'None'}]
[{'key': 'first_next_item', 'value': 'gft'}]
[{'key': 'first_next_date', 'value': '30-04-2020'}]
['pmd', 'gft ', 'restafval', 'papier']
```

**Comment by xirixiz** [comment]
Hmm, but when I add it to my hass config, I also get "Geen". I`ll reopen the issue and investigate further.

**Comment by xirixiz** [comment]
Found it! See the nasty space after gft...

**Comment by tomlevels** [comment]
Great you found the issue! I already expected it had something to do with the space...

**Comment by xirixiz** [comment]
Fixed in 4.1.9
